test/broadway_test.exs

```diff
@@ -23,6 +23,16 @@ defmodule BroadwayTest do
     end
   end
 
+  defmodule AckerWithConfigure do
```

Comment:
Instead of adding this Acker... we should perhaps use the caller acknowledger. This way we test both the functionality and the fact the caller acknowledger works.

Reply:
Done!